README.rst

```diff
@@ -98,45 +98,48 @@ Documentation (WIP)
 
 The `full documentation <https://serenata_toolbox.readthedocs.io>`_ is still a work in progress. If you wanna give us a hand you will need `Sphinx <http://www.sphinx-doc.org/>`_:
 
-::
+.. code-block:: bash
 
   $ cd docs
   $ make clean;make rst;rm source/modules.rst;make html
 
 Contributing
 ------------
 
-Within your `virtualenv <https://virtualenv.pypa.io/en/stable/>`_:
+Start by cloning the repository and installing the dependencies. You should create a development environment with `virtualenv <https://virtualenv.pypa.io/en/stable/>`_ by running:
 
-::
+.. code-block:: bash
 
   $ git clone https://github.com/okfn-brasil/serenata-toolbox.git
+  $ cd serenata-toolbox
+  $ virtualenv .env
+  $ source .env/bin/activate
```

Comment:
Usually I don't include virtualenv options because different setups might have different ways to crate virtualenvs (`virtualenv`, `python -m venv`, `pipenv`, `mkvirtualenv` from `virtualenv-wrapper`). **If** you think it's really important the default for Python 3.6 is `python -m venv` I guess (but need to be checked in the docs).

Reply:
IMHO I think it's important to describe _all_ the commands necessary to run the application. Create and activate the dev environment is very basic and any long-time developer would do that without hesitating, but I'm thinking about the beginners who want to contribute.

Reply:
What should I do? You have the final call.

Reply:
Just checked [the docs](https://docs.python.org/3/library/venv.html#creating-virtual-environments): the command would be `python -m venv .env`, not `virtualenv .env` — so I'll keep the _requested changes_ status.

But more important than this detail is what follows:

Anyway, I'm also _thinking about the beginners who want to contribute_ — if you think otherwise you're getting me completely wrong. However my point is: if they just copy and past commands without any understanding of what's going on this is not _necessarily_ the best way to bring them onboard. If our commands happen to not work for their system, this generates noise and discourage them to move on. 

I do prefer to indicate _you have to create a virtualenv_ and let them do it the way they have already learned, or do their own research according to their system and needs. So we avoid commands that might not work for some users. 

Wanna any use cases where this approach is better than `virtualenv .env` plus `source .env/bin/activate`? Here we go:  I was told calling the virtualenv `.env` might not work in some Windows version; but as a matter of fact `source .env/bin/activate` does not work on Windows at all! And [49% of Python developers use Windows](https://www.jetbrains.com/research/python-developers-survey-2017/#developer-profile) – yet, 17% use other OS than macOS, Linux and Windows.

So I do understand you're willing _to describe all the commands_ but I don't think that's feasible (or, if so, if it's desirable because our docs would be too verbose full of special cases). That's why in the docs I tend to focus on what is specific to the application I'm documenting and let the language- and environment-specific instructions for users to do their own research according to their context and needs.

Reply:
What about linking to the official Python docs on `venv`?

---

Within your [`venv`](https://docs.python.org/3/library/venv.html#creating-virtual-environments) clone the repository and install the package for development:

```sh
$ git clone https://github.com/okfn-brasil/serenata-toolbox.git
$ cd serenata-toolbox
$ python setup.py develop
```
---

What do you think?